provisioning_templates/snippet/kickstart_ifcfg_generic_interface.erb

```diff
@@ -40,7 +40,7 @@ DEFROUTE=<%= primary %>
 <%-   if !@dhcp && @subnet -%>
 <%-     @subnet.dns_servers.each_index { |index| -%>
```

Comment:
Would it make sense to use `do` here since it's multi-line?
```suggestion
<%-     @subnet.dns_servers.each_index do |index| -%>
```

Reply:
You actually bring the bug back with this suggestion :-) Anyway, I am fine with this.

Reply:
I knew that, but didn't know how else to express it and (correctly) guessed you could figure it out :)